indico/modules/events/abstracts/forms.py

```diff
@@ -74,6 +74,29 @@ def validate_contrib_type_required(self, field):
             raise ValidationError(_('The event has no contribution types defined.'))
 
 
+class AbstractReviewingSettingsForm(IndicoForm):
+    """Settings form for abstract reviewing"""
+
+    num_answers = IntegerField(_('Number of choices'), [NumberRange(min=2), DataRequired()],
```

Comment:
I think `num_answers` is wrong naming (inherited from legacy IIRC). It's not how many answers you can give to a question, but how many choices are given as an answer (as in the label and description).